src/components/largePipelineWarning/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleDisplayLargeGraph } from '../../actions/graph';
+import './largePipelineWarning.css';
+
+export const LargePipelineWarning = ({
+  nodeCount,
+  onToggleDisplayLargeGraph
+}) => {
+  return (
+    <div className="warning">
```

Comment:
Check with @richardwestenra on this, but I think all classnames in this project are prefixed with `pipeline-`

Reply:
Thanks for the suggestion @bru5 - updated naming to include the prefix